aws-cli/bash-linux/s3/bucket-lifecycle-operations/bucket-operations-test.sh

```diff
@@ -16,11 +16,7 @@
 
 METADATA-DO-NOT-REMOVE
 
-function pause {
-    if [[ $interactive = true ]]; then
-        read -p "Press ENTER to continue..."
-    fi
-}
+source ./general.sh
```

Comment:
Why are you sourcing this file? I don't see you sourcing it in any other file. Or are you planning on replacing a chunk of bucket-operations.sh with this source directive?

Reply:
This was my very first example, so there are no other scripts to source it from YET.  :-)

That file has general purpose functions that I intend on using across many scripts and tests. I'll be including a copy with most example tests. I also think it demonstrates good best practice of factoring out common functions to avoid duplicating code.

Reply:
Because this is my very first example. I plan on using this file across many of the examples I create and will simply paste it in and source it in each. Shows best practice of refactoring common code into reusable modules.